include/oneapi/internal/kernel_scheduler_sycl.h

```diff
@@ -181,14 +337,45 @@ class OpenClKernel : public Base, public KernelIface
 
     ExecutionTargetId getTarget() const { return _executionTarget; }
 
-    const OpenClKernelRef & getRef() const { return _clKernelRef; }
+    virtual cl::sycl::kernel toSycl(const cl::sycl::context & _ctx) const {}
```

Comment:
Nothing to return?

Reply:
If this method isn't expected to be called it should fire a error maybe?

Reply:
fixed